src/index.d.ts

```diff
@@ -72,6 +72,12 @@ export function withTranslation(
   component: React.ComponentType<P>,
 ) => React.ComponentType<Omit<P, keyof WithTranslation>>;
 
+export interface I18nextProviderProps {
+  i18n: i18next.i18n;
+}
+
+export const I18nextProvider: React.ComponentClass<I18nextProviderProps>;
```

Comment:
Shouldn't this be `React.FunctionComponent<I18nextProviderProps>` or `React.FC<I18nextProviderProps>`? Because basically the component is a function component, not a class one.

Reply:
Then it should be React.FunctionComponent